pkgs/development/tools/continuous-integration/gocd-agent/default.nix

```diff
@@ -2,12 +2,12 @@
 
 stdenv.mkDerivation rec {
   name = "gocd-agent-${version}-${rev}";
-  version = "19.3.0";
-  rev = "8959";
+  version = "21.2.0";
+  rev = "12498";
 
   src = fetchurl {
     url = "https://download.go.cd/binaries/${version}-${rev}/generic/go-agent-${version}-${rev}.zip";
-    sha256 = "1nirdv82i8x4s1dyb0rmxldh8avappd4g3mbbl6xp7r7s0drcprp";
+    sha256 = "105e38058cb918bf0ff2c8a2458a911fddf2ff347d8e9ba7c9107972d58439f2";
```

Comment:
Why are we repeating this?

Reply:
I am honestly not sure what exactly you're referring to here; Both the URL as well as the hash differ from the `gocd-server` ones; While I'm assuming the same feedback as with that file still applies here, its not a repetition as far as I can tell